include/z64environment.h

```diff
@@ -70,6 +70,27 @@ typedef enum WeatherMode {
     /* 5 */ WEATHER_MODE_HEAVY_RAIN // scene must define settings for light config 4
 } WeatherMode;
 
+typedef enum {
+// Clear Sky
+    SKYBOX_INDEX_CLEAR_SUNRISE,
+    SKYBOX_INDEX_CLEAR_DAY,
+    SKYBOX_INDEX_CLEAR_SUNSET,
+    SKYBOX_INDEX_CLEAR_NIGHT,
+// Cloudy Sky
+    SKYBOX_INDEX_CLOUDY_SUNRISE,
+    SKYBOX_INDEX_CLOUDY_DAY,
+    SKYBOX_INDEX_CLOUDY_SUNSET,
+    SKYBOX_INDEX_CLOUDY_NIGHT,
```

Comment:
Wondering if we have a better descriptor than just "index" for this concept in general. Ill take a closer look at this and try to think about it more, but it makes reading all of the code pretty clunky to me.
Dont think "type" works cause this isnt necessarily the skybox type. But looking for something along those lines